garak/generators/litellm.py

```diff
@@ -105,13 +106,19 @@ class LiteLLMGenerator(Generator):
         "skip_seq_start",
         "skip_seq_end",
         "stop",
+        "suppressed_params",
     )
 
     def __init__(self, name: str = "", generations: int = 10, config_root=_config):
         self.name = name
         self.api_base = None
         self.provider = None
         self._load_config(config_root)
+        
+        # Ensure suppressed_params is a set for efficient lookup
+        if hasattr(self, 'suppressed_params') and not isinstance(self.suppressed_params, set):
+            self.suppressed_params = set(self.suppressed_params)
```

Comment:
The existence check here is not _required_ as `DEFAULT_PARAMS` will be projected on self even if provided as `None`/`null` values in the passed configuration. 

The type check also seems reasonable to skip as yaml and json loaders will often be a list as I suspect users are unlikely to add the extra syntax in yaml for a set and json does not support set by default.

Consider:
```suggestion
        self.suppressed_params = set(self.suppressed_params)
```

Enforcement wise I believe this will would create the same result and allow `set()` to raise if provided a value that cannot be coalesced into a `set`.  This is would still be imperfect type checking however the project has not yet settled on a pattern for type check enforcement in plugins.  There are some ideas floating around about considering type providing hints on `DEFAULT_PARAMS` in the future.